src/test/regress/sql/schemas.sql

```diff
@@ -0,0 +1,2 @@
+SELECT nspname FROM pg_namespace WHERE nspname LIKE 'test_schema__' ORDER BY nspname;
+SELECT * FROM test_schema_3.test_table;
```

Comment:
You'll need to update the schema name if you add a `ALTER TABLE .. SET SCHEMA` command that I suggested in another comment.